avocado/core/jobdata.py

```diff
@@ -30,7 +30,7 @@
 CONFIG_FILENAME = 'config'
 TEST_REFERENCES_FILENAME = 'test_references'
 TEST_REFERENCES_FILENAME_LEGACY = 'urls'
-MUX_FILENAME = 'multiplex'
+VARIANTS_FILENAME = 'multiplex'
```

Comment:
Right now nothing. I think we should change it when we come up with the definite solution of the multiplex jobdata storage (I mean the implementation not based on pickle)